source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -2065,6 +2083,22 @@ TEST_F(NiDAQmxDriverApiTests, LoadedVoltageTask_ReadAIData_ReturnsDataInExpected
   EXPECT_DATA_IN_RANGE(read_response.read_array(), AI_MIN, AI_MAX);
 }
 
+TEST_F(NiDAQmxDriverApiTests, BridgeOffsetNullingCal_Succeeds)
+{
+  auto response = PerformBridgeOffsetNullingCalExResponse{};
+  auto status = perform_bridge_offset_nulling_cal_ex(response);
+
+  EXPECT_SUCCESS(status, response);
+}
+
+TEST_F(NiDAQmxDriverApiTests, ThrmcplLeadOffsetNullingCal_Succeeds)
+{
+  auto response = PerformThrmcplLeadOffsetNullingCalResponse{};
+  auto status = perform_thrmcpl_lead_offset_nulling_cal(response);
+
+  EXPECT_SUCCESS(status, response);
```

Comment:
These tests need to create a strain or thermocouple channel.

SetUp() and TearDown() already create/destroy a DAQmx task.

Reply:
Thanks for pointing this out, I have added helper functions to create a bridge and thermocouple channels respectively. However I am unable to test it for now because I'm facing build issues, I am just eyeballing it and will test it once I am able to.